charger/daheimladen.go

```diff
@@ -0,0 +1,193 @@
+package charger
+
+import (
+	"fmt"
+	"net/http"
+
+	"github.com/evcc-io/evcc/api"
+	"github.com/evcc-io/evcc/charger/daheimladen"
+	"github.com/evcc-io/evcc/util"
+	"github.com/evcc-io/evcc/util/request"
+	"golang.org/x/oauth2"
+)
+
+// DaheimLaden charger implementation
+type DaheimLaden struct {
+	*request.Helper
+	stationID     string
+	connectorID   int32
+	idTag         string
+	token         string
+	transactionID int32
+}
+
+func init() {
+	registry.Add("daheimladen", NewDaheimLadenFromConfig)
+}
+
+// NewDaheimLadenFromConfig creates a DaheimLaden charger from generic config
+func NewDaheimLadenFromConfig(other map[string]interface{}) (api.Charger, error) {
+	cc := struct {
+		Token     string
+		StationID string
+		IDTag     string
```

Comment:
Where does the user get the idtag? Could we instead create a unique id ourselves?

Reply:
I initially thought a user would punch in the card number from one of the RFID cards that we provide, but since the charging session will be started by evcc how about we keep the `IDTag` as "evcc"?

Reply:
I've added it as `evcc` for now

Reply:
Good idea and less user config, yet obvious choice 👍🏻